docs/biovalidator_benchmarks/2023.01.23_benchmarks/json_docs/analysis_valid-1.json

```diff
@@ -0,0 +1,152 @@
+{
+    "data": {
+        "analysisAttributes": [
+            {
+                "tag": "sub_project",
+                "value": "CB4-1UD"
+            }
+        ],
+        "analysisFiles": [
+            {
+                "checksumMethod": "MD5",
+                "encryptedChecksum": "c6779ec2960296ed9a04f08d67fa4415",
+                "filename": "sequence_variation.vcf.gpg",
+                "filetype": "VCF",
+                "unencryptedChecksum": "9a4249e4b0538b9789c5d7f25d6342f0"
+            },
+            {
+                "checksumMethod": "MD5",
+                "encryptedChecksum": "d92463599ed4548cfa4fff55c1b48d7e",
+                "filename": "analysis_script.py.gpg",
```

Comment:
Why is the python script GPG encrypted?

Reply:
I think not just the python file, but also the vcf with a ``.cpg`` suffix is wrong: we don't normally include the suffix in the submitted filename. I'll fix it, good call 👍